### ISSUE TITLE
Suggestion: User Interface

### ISSUE THREAD

**Issue by kinzozs** [Just2good/TFT-Overlay#25]
Hello, if its not that much of a problem can you add a skin menu to select back the old UI, the new one seems to messy to me, the old one was simple and all in one place

**Comment by Just2good** [comment]
Will do.
A lot of people preferred the old skin so I was planning on it.

**Comment by Jrayp** [comment]
Yeah this new design is a step in the wrong direction imo. The old design was very intuitive, More clicks for the same functionality is never a good thing
